client/src/App.js

```diff
@@ -11,6 +13,12 @@ const App = () => (
 		<Route path="/about/this/site">
 			<About />
 		</Route>
```

Comment:
Hi Laura, do we have to still keep the About page, if so, what is your opinion to fill in?

Reply:
Not sure if we need the About page, as I have added an intro on the Home page that explains the overall concept; but let's keep the route for now and maybe we can add it to the Footer.